From foo@baz Fri Apr 17 09:51:01 PDT 2009
Date: Fri, 17 Apr 2009 09:51:01 -0700
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Staging: comedi: set PCI and PCMCIA defines

The comedi drivers are looking for CONFIG_COMEDI_PCI and
CONFIG_COMEDI_PCMCIA, not the current config items.  This creates a
define so that things build properly when these options are selected.
Long term goal is to fix up the drivers to not need any defines.

Cc: Frank Mori Hess <fmhess@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/comedi/comedidev.h |   13 +++++++++++++
 1 file changed, 13 insertions(+)

--- a/drivers/staging/comedi/comedidev.h
+++ b/drivers/staging/comedi/comedidev.h
@@ -525,4 +525,17 @@ void comedi_usb_auto_unconfig(struct usb
 
 #include "comedi_rt.h"
 
+#ifdef CONFIG_COMEDI_PCI_DRIVERS
+	#define CONFIG_COMEDI_PCI
+#endif
+#ifdef CONFIG_COMEDI_PCI_DRIVERS_MODULE
+	#define CONFIG_COMEDI_PCI
+#endif
+#ifdef CONFIG_COMEDI_PCMCIA_DRIVERS
+	#define CONFIG_COMEDI_PCMCIA
+#endif
+#ifdef CONFIG_COMEDI_PCMCIA_DRIVERS_MODULE
+	#define CONFIG_COMEDI_PCMCIA
+#endif
+
 #endif /* _COMEDIDEV_H */
